product_brand_purchase/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+#. Go to *Purchase > Reporting > Purchase*.
+#. You can now group by brand.
```

Comment:
#. "Product brands" menu is now accessible also under *Purchase > Configuration > Products*